qtkeychain.qbs

```diff
@@ -0,0 +1,139 @@
+import qbs
+
+Product
+{
+    name: "qtkeychain"
+    type: buildStatic ? "staticlibrary" : "dynamiclibrary"
+    targetName: "qtkeychain"
+
+    property bool buildTranslations: false
+    property bool buildStatic: false
+    property bool libsecretSupport: false
+
+    Depends { name: "Quarks" }
```

Comment:
This shouldn't depend on Quarks is what I think.
Just like the SortFilterProxyModel (which needs a cleanup too).
We can just add this as a submodule in hyperion.

Reply:
Ah, I see. So a direct submodule of hyperion. That would make sense, since we are not interested in having this on non-hyperion builds.

Reply:
Ok @saidinesh5, I've removed the quarks stuff: this `Depends` and the conditional compiling on `quarks` enabled flag.